src/Core/Services/OrganizationService.cs

```diff
@@ -51,5 +54,19 @@ public async Task ClearAllAsync(string userId)
         {
             await _stateService.SetOrganizationsAsync(null, userId);
         }
+
+        public async Task<string> GetClaimedOrganizationDomainAsync(string userEmail)
+        {
+            try
+            {
+                var response = await _apiService.GetOrgDomainSsoDetails(userEmail);
+                return response.OrganizationIdentifier;
+            }
+            catch (System.Exception ex)
+            {
+                // api throws 404 if there is no domain claimed
+                return string.Empty;
+            }
```

Comment:
Is it ok that any exception returns empty string? or should it only check when the API exception is because of a 404 code?

Reply:
If something happens we just want to ignore it and let the user be in the screen to continue to login manually. What i missed was logging it, just in case.